core/server/data/migration/fixtures/005/05-transform-dates-into-UTC.js

```diff
@@ -0,0 +1,132 @@
+var config = require('../../../../config'),
+    models = require(config.paths.corePath + '/server/models'),
+    sequence = require(config.paths.corePath + '/server/utils/sequence'),
+    moment = require('moment'),
+    Promise = require('bluebird'),
+    messagePrefix = 'Transform into UTC Dates: ';
+
+/**
+ * there is no chance to calculate UTC dates 100% accurate from DB values because of DST
+ */
+module.exports = function transformDatesIntoUTC(options, logger) {
+    if (ServerTimezoneOffset === 0) {
+        logger.info('Your Application uses UTC, skip!');
+        return Promise.resolve();
+    }
+
+    return sequence([
+        function () {
+            return models.Posts.forge().fetch().then(function (result) {
+                if (result.models.length === 0) {
+                    logger.warn(messagePrefix + 'No Posts found');
+                    return;
+                }
+
+                return Promise.map(result.models, function mapper(post) {
+                    post.set('published_at', moment(post.get('published_at')).add(ServerTimezoneOffset, 'minutes').toDate());
```

Comment:
For this migration, I'd switch `global.Date` back to the native constructor first. Then `moment(post.get('published_at')).utc()` would give us the time properly converted to UTC from the local TZ with respect to DST. Of course, I must admit this looks somewhat crazy:

```js
convertToUtc(value) {
  var overriddenDate = global.Date;
  global.Date = NativeDate;
  value = moment(value).utc();
  global.Date = overriddenDate;
  return value.toDate();
}
```